bot/scrypt_tunes.py

```
@@ -239,51 +251,226 @@ async def np_command(self, ctx):
+                }
+                r = req.post("https://api.twitch.tv/helix/eventsub/subscriptions", headers=headers, json=body, timeout=10)
+                if r.status_code not in (200, 202):
+                    logging.error("EventSub subscription create failed (status=%s): %s", r.status_code, r.text)
+                    return
+                logging.info("EventSub subscription created for Channel Points redemptions")
+
+                async for msg in ws:
+                    if msg.type == aiohttp.WSMsgType.TEXT:
+                        try:
+                            data = msg.json() if hasattr(msg, "json") else json.loads(msg.data)
+                        except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)